### PR TITLE
Adding triggers to MicrosoftGraphSecurity connector

### PR DESCRIPTION
**Pull request by Mounika-Chillamcherla**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### CODE REVIEW

**Comment by Mounika-Chillamcherla** [comment]
Hey., Can someone check this PR ??

**Comment by mamurshe** [comment]
@Mounika-Chillamcherla I am looking at it.

File: connectors/MicrosoftGraphSecurity/apiProperties.json [hunk 1]
```diff
@@ -1,27 +1,76 @@
 {
   "properties": {
-    "capabilities": [
-      "actions"
```
Comment (dmkuby) [inline connectors/MicrosoftGraphSecurity/apiProperties.json]: Is it not true anymore? Should it be extended to ["actions","triggers"] instead?

Comment (Mounika-Chillamcherla) [reply connectors/MicrosoftGraphSecurity/apiProperties.json]: Updated it to ["actions", "triggers"]

File: connectors/MicrosoftGraphSecurity/apiProperties.json [hunk 2]
```diff
@@ -1,27 +1,79 @@
...
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "scopes": [
+            "User.Read"
+          ]
         },
         "type": "oauthSetting"
+      },
+      "token:TenantId": {
+        "metadata": {
+          "sourceType": "AzureActiveDirectoryTenant"
```
Comment (ShyamSunder90) [inline connectors/MicrosoftGraphSecurity/apiProperties.json]: Is this intended change?

File: connectors/MicrosoftGraphSecurity/apiProperties.json [hunk 3]
```diff
@@ -1,27 +1,79 @@
...
+      "triggers"
     ],
     "connectionParameters": {
       "token": {
         "oAuthSettings": {
           "clientId": "{{replace_with_your_client_id}}",
           "customParameters": {
+            "loginUri": {
+              "value": "https://login.windows.net"
+            },
```
Comment (ShyamSunder90) [inline connectors/MicrosoftGraphSecurity/apiProperties.json]: Is this intended?

File: connectors/MicrosoftGraphSecurity/apiProperties.json [hunk 4]
```diff
@@ -1,27 +1,79 @@
...
             "resourceUri": {
-              "value": "https://graph.microsoft.com"
+              "value": "https://graph.microsoft.com/"
+            },
+            "tenantId": {
+              "value": "common"
```
Comment (ShyamSunder90) [inline connectors/MicrosoftGraphSecurity/apiProperties.json]: Is this intended?

File: connectors/MicrosoftGraphSecurity/apiProperties.json [hunk 5]
```diff
@@ -1,27 +1,79 @@
...
             }
           },
           "identityProvider": "aad",
           "properties": {
             "IsFirstParty": "False"
           },
           "redirectMode": "Global",
-          "redirectUrl": "https://global.consent.azure-apim.net/redirect"
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "scopes": [
```
Comment (ShyamSunder90) [inline connectors/MicrosoftGraphSecurity/apiProperties.json]: Is this intended?

**Pull request merged**